lib/i18n/tasks/ar/attribute.rb

```diff
@@ -10,6 +10,7 @@ class << self
           # 
           def names
             result = {}
+            Rails.application.eager_load! if Rails.env == 'development' rescue true
```

Comment:
Line is too long. [83/80]<br>Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.